floodfill_selection/volume_selection.py

```diff
@@ -343,9 +350,7 @@ def on_mouse_move(self, event):
                 height = event.pos[1] - self.selection_origin[1]
                 drag_distance = math.sqrt(width**2+height**2)
                 canvas_diag = math.sqrt(self.canvas.size[0]**2 + self.canvas.size[1]**2)
-                # normalize the threshold between max and min value
-                normalize = (np.max(self.vol_data) - np.min(self.vol_data))/canvas_diag
-                self.draw_floodfill_visual(drag_distance*normalize)
+                self.draw_floodfill_visual(drag_distance / canvas_diag)
```

Comment:
Normalizing just by the diagonal means we'll always end up with a value between 0 and 1 and makes it easy to transform to the range 1.01->101 later. Now that the flood fill uses a relative value, this is fine.